docs/how-to-guides/themes/create-block-theme.md

```diff
@@ -783,9 +780,7 @@ Since the theme has custom padding enabled, you can add `padding` within the `sp
 ### Template parts
 
 In the templeParts section, assign the two template parts that you created to their template areas.
-Add two keys:
--`name`, the file name of the template part file without the file extension,
--`area`, the name of the template area.
+Add two keys: -`name`, the file name of the template part file without the file extension, -`area`, the name of the template area.
```

Comment:
Add the third key, which is is `title`, the visible name in the editor.

Reply:
Added